README.md

```diff
@@ -12,7 +12,7 @@ or
 
 ```
 ## at Linux
-curl -slO https://github.com/fujiwara/nssh/releases/download/v0.0.1/nssh-v0.0.1-linux-amd64.zip
+curl -slLO https://github.com/fujiwara/nssh/releases/download/v0.0.1/nssh-v0.0.1-linux-amd64.zip
```

Comment:
Sorry, `-l` option was typo of `-L`.
Cloud you replace `slLO` to `sLO` ?

Reply:
OK. I fix it.